Application/EdFi.Ods.SwaggerUI/wwwroot/index.js

```diff
@@ -0,0 +1,169 @@
+sections.Composites.color = 'green-text'
+sections.Other.color = 'orange-text'
+sections.Profiles.color = 'red-text'
+sections.Resources.description = [
+  'Resources are the primary entities that most API client applications work with on a regular basis. Students, staff, education organizations, and their related entities are maintained using this area of the API',
+]
+sections.Composites.description = [
+  'Composites are read-only combinations of resources that address specific business use cases. Using composites reduces the number of calls that an API client application must make to retrieve resources for these use cases.',
+  'For example, the Enrollment composite can return all the students in a given class section or all students associated with a particular staff member, and similar student association-related information in a single call. Platform hosts may add their own composite resources.',
+]
+sections.Other.description = [
+  'The other area of the API contains functional resources. These resources may be enabled or disabled by the ODS / API platform host.',
+  'The identity management API is used to access a native identity system if one exists in the enterprise. The endpoints provide client applications with the ability to find and (optionally) create unique person identifiers that can be used with Student, Staff, and Parent resources.',
+]
+sections.Profiles.description = [
+  'Profiles are used by platform hosts to restrict access to properties of a resource. Resource properties may be read-write, read-only, or unavailable. When API client applications use a profile to access resources, they are limited to a subset of the properties available on the underlying resource.',
+]
+
+function hideProgress() {
+  var progress = document.getElementById('progress')
+  progress.classList.add('hide')
+
+  var progressDescription = document.getElementById('progressDescription')
+  progressDescription.classList.add('hide')
+}
+
+function showVersion(json) {
+  var version = document.getElementById('version')
+  version.textContent = 'v' + json.informationalVersion
+  version.className = version.className.replace('hide', '') // IE11 support
+  version.classList.remove('hide')
+
+  return json
+}
+
+function showPageDescription() {
+  var pageDescription = document.getElementById('pageDescription')
+  pageDescription.className = pageDescription.className.replace('hide', '') // IE11 support
+  pageDescription.classList.remove('hide')
+}
+
+function showError(message) {
+  var errorDescription = document.getElementById('errorDescription')
+  errorDescription.innerHTML += '<p>' + message + '</p>'
+  errorDescription.className = errorDescription.className.replace('hide', '') // IE11 support
+  errorDescription.classList.remove('hide')
+}
+
+// maps the json data from the metadata url to the sections variable
+function mapSections(json) {
+  json.forEach(function (data) {
+    var prefix = data.Prefix === '' ? 'Resources' : data.Prefix
```

Comment:
lets create a ticket to address this in the future when the netcore changes come in. Note the red in the image are the changes we will need to do.

![image](https://user-images.githubusercontent.com/288205/91463162-aca8e900-e850-11ea-93d9-7313690fd2d5.png)